src/EnergyPlus/DXCoils.cc

```diff
@@ -2979,895 +2979,978 @@ void GetDXCoils(EnergyPlusData &state)
+                ErrorsFound = true;
             }
-        }
 
-        if (!lAlphaBlanks(12)) {
-            thisDXCoil.HCapFAirFlow = GetCurveIndex(state, Alphas(12));
-            if (thisDXCoil.HCapFAirFlow == 0) {
+            cFieldName = "Maximum Ambient Temperature for Crankcase Heater Operation";
+            thisDXCoil.MaxOATCrankcaseHeater =
+                s_ip->getRealFieldValue(fields, schemaProps, "maximum_ambient_temperature_for_crankcase_heater_operation"); // Numbers(12);
+            if (thisDXCoil.MaxOATCrankcaseHeater < 0.0) {
                 ShowSevereError(state, format("{}{}=\"{}\", invalid", RoutineName, CurrentModuleObject, thisDXCoil.Name));
-                ShowContinueError(state, format("...not found {}=\"{}\".", cAlphaFields(12), Alphas(12)));
+                ShowContinueError(state, format("...{} must be >= 0 {{C}}.  entered value=[{:.1T}].", cFieldName, thisDXCoil.MaxOATCrankcaseHeater));
                 ErrorsFound = true;
-            } else {
-                // Verify Curve Object, only legal types are Cubic or Quadratic
-                ErrorsFound |= Curve::CheckCurveDims(state,
-                                                     thisDXCoil.HCapFAirFlow, // Curve index
-                                                     {1},                     // Valid dimensions
-                                                     RoutineName,             // Routine name
-                                                     CurrentModuleObject,     // Object Type
-                                                     thisDXCoil.Name,         // Object Name
-                                                     cAlphaFields(12));       // Field Name
+            }
 
-                if (!ErrorsFound) {
-                    checkCurveIsNormalizedToOne(state,
-                                                std::string{RoutineName} + CurrentModuleObject,
-                                                thisDXCoil.Name,
-                                                thisDXCoil.HCapFAirFlow,
-                                                cAlphaFields(12),
-                                                Alphas(12),
-                                                1.0);
+            cFieldName = "Crankcase Heater Capacity Function of Temperature Curve Name"; // cAlphaFields(9)
+            std::string const whPLFCurveName =
+                s_ip->getAlphaFieldValue(fields, schemaProps, "crankcase_heater_capacity_function_of_temperature_curve_name");
+            // Coil:WaterHeating:AirToWaterHeatPump:Wrapped
+            if (!whPLFCurveName.empty()) {
+                thisDXCoil.CrankcaseHeaterCapacityCurveIndex = Curve::GetCurveIndex(state, whPLFCurveName);
+                if (thisDXCoil.CrankcaseHeaterCapacityCurveIndex == 0) { // can't find the curve
+                    ShowSevereError(state, format("{} = {}:  {} not found = {}", CurrentModuleObject, thisDXCoil.Name, cFieldName, whPLFCurveName));
```

Comment:
There is also an error wrapper for missing objects. 

```cpp 
ShowSevereItemNotFound(state, eoh, cFieldName, whPLFCurveName);`
```

Reply:
Done.